src/Registration/Common/MutualInformation/MutualInformation.ipynb

```diff
@@ -0,0 +1,703 @@
+{
```

Comment:
<p>We can replace these cells with the shorter, more Pythonic:</p><p> </p><pre class="ql-syntax" spellcheck="false">fixed_normalized = itk.normalize_image_filter(fixed_image)
fixed_smoothed = itk.discrete_gaussian_image_filter(fixed_normalized, variance=2.0)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKExamples/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>